app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/warning_govuk.scala.html

```diff
@@ -1,18 +1,30 @@
-
-
 @*
-* Copyright 2021 HM Revenue & Customs
+* Copyright 2022 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
 *
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
 *@
-@import play.api.i18n.Messages
+
+@import play.twirl.api.Html
 @import uk.gov.hmrc.govukfrontend.views.html.components._
 
 @this(govukWarningText : GovukWarningText)
 
-@(warningText: String)(implicit messages: Messages)
-
+@(warningText: String)
 
 @govukWarningText(WarningText(
 iconFallbackText = "Warning",
-content = Text(messages(warningText))
+content = HtmlContent(Html(s"""
+    |<strong class="bold-small" id="warning">
```

Comment:
Can this use the govukWarningText component?

Reply:
GovukWarningText is being used here, this is being passed into the content parameter.

Reply:
Ah yes sorry missed that :)